stan/model.py

```diff
@@ -233,6 +233,45 @@ def is_iteration_or_elapsed_time_logger_message(msg: simdjson.Object):
         except KeyboardInterrupt:
             pass
 
+    def log_prob(self, unconstrained_parameters: typing.Sequence[float], adjust_transform: bool = True):
+        """Calculate the log probability of a set of unconstrained parameters.
+
+        Arguments:
+            unconstrained_parameters: A sequuence of unconstrained parameters.
+            adjust_transform: Apply jacobian adjust transform.
+
+        Returns:
+            Number: The log probability of the unconstrained parameters.
+
+        Notes:
+            The unconstrained parameters are passed to the log_prob_propto
+            function in stan::model.
+
+        """
+        assert isinstance(self.data, dict)
+        unconstrained_parameters = {"upar": unconstrained_parameters}
+        unconstrained_parameters = _make_json_serializable(unconstrained_parameters)
+
+        payload = {"data": self.data}
+        payload["unconstrained_parameters"] = unconstrained_parameters["upar"]
+        payload["adjust_transform"] = adjust_transform
+
+        async def go():
+            io = ConsoleIO()
+            async with stan.common.httpstan_server() as (host, port):
+                log_prob_url = f"http://{host}:{port}/v1/{self.model_name}/log_prob"
+                async with aiohttp.request("POST", log_prob_url, json=payload) as resp:
+                    response_payload = await resp.json()
+                    if resp.status != 200:
+                        raise RuntimeError(response_payload)
+                    log_prob = (await resp.json())["log_prob"]
+            return log_prob
+
+        try:
+            return asyncio.run(go())
+        except KeyboardInterrupt:
+            pass
```

Comment:
```suggestion
        return asyncio.run(go())
```
Let's remove the KeyboardInterrupt guard. This function is going to return so fast that nobody is going to have time to hit Control-C.